### PR TITLE
layout and index page changes

### PR DESCRIPTION
**Pull request by camallen**
provide useful links on landing page and make the logo centered and bigger.

### CODE REVIEW

File: app/views/home/index.html.erb [hunk 1]
```diff
@@ -1,5 +1,23 @@
...
+  </div>
+</div>
+<div class="home-landing">
+  <div class="container">
+    <div class="row">
+      <div class="col-sm-2 col-sm-offset-4">
+        <%= link_to "Sign In", new_user_session_path, class: "btn btn-success"%>
+      </div>
+      <div class="col-sm-2">
+        <%= link_to "Sign Out", destroy_user_session_path, class: "btn btn-danger"%>
```
Comment (edpaget) [inline app/views/home/index.html.erb]: This needs to have a `method: :delete` or the sign out button will raise a 404 error.

Comment (camallen) [inline app/views/home/index.html.erb]: yep - my bad.

Comment (edpaget) [inline app/views/home/index.html.erb]: I fixed it locally. I'm just going to commit that and merge into master

Comment (camallen) [inline app/views/home/index.html.erb]: cool - please do.

**Pull request merged**